recipes/pymodsecurity/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pymodsecurity" %}
+{% set version = "0.0.1" %}
+{% set build_number = 0 %}
+{% set hash_value = "e9570494282b71172f909cb3ecfd4b7b66fa29a3141439192a48b40939893495" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash_value }}
+
+build:
+  number: {{ build_number }}
+  skip: true  # [not linux]
+  script_env:
```

Comment:
I'm not sure that the `script_env` section is necessary. I would think that the `compiler`'s would take care of this.